docs/visualizing-your-sql-queries/plotting-with-seaborn.md

```diff
@@ -1,14 +1,561 @@
+
+A few examples denoting this distinction are shown below.
+
+### Axes-level
+
+Suppose we want to identify whether gentrification increases the average salary of two regions, using the data we downloaded above. We first save our [CTE (Common Table Expression)](https://ploomber-sql.readthedocs.io/en/latest/advanced-querying-techniques/ctes) named `levels_example` that takes in the columns, `average_salary`, `ratio_of_urban_inhabitants`, and `region`, and filters for two regions, 'central Bohemia' and 'east Bohemia', from the `s1.district` table.
+
+```{code-cell} ipython3
+%%sql --save levels_example
+SELECT average_salary, ratio_of_urban_inhabitants, region
+FROM s1.district
+WHERE region IN ('central Bohemia', 'east Bohemia');
+```
+
+The result of the CTE is saved as a pandas `DataFrame()`:
+
+```{code-cell} ipython3
+result = %sql SELECT * FROM levels_example
+df = result.DataFrame()
+```
+
+You can determine what is returned using Python's `type()` function:
+
+```{code-cell} ipython3
+import seaborn as sns   # noqa: E402
+import matplotlib.pyplot as plt   # noqa: E402
+
+plt.rcParams["figure.dpi"] = 300  # high resolution
+
+scatter_plt = sns.scatterplot(
+    data=df, x="ratio_of_urban_inhabitants", y="average_salary", hue="region"
+)
+print(type(scatter_plt))
+```
+
+Notice that the `sns.scatterplot()` function returns a Matplotlib `Axes` object, a single plot that is inclusive of the data from both regions. Other seaborn functions, including `regplot`, `boxplot`, `kdeplot`, and many others, also return Matplotlib `Axes` objects. Therefore, we can use various Matplotlib axes [commands](https://seaborn.pydata.org/generated/seaborn.axes_style.html) to modify the Seaborn figure.
+
+### Figure-level
+
+On the other hand, we can show that `sns.relplot()` returns a `FacetGrid` object, creating separate plots for the regions:
+
+```{code-cell} ipython3
+facet_plt = sns.relplot(
+    data=df, x="ratio_of_urban_inhabitants", y="average_salary", col="region"
+)
+print(type(facet_plt))
+```
+
+Other Figure-level seaborn functions include `catplot`, `displot`, `pairplot`, and `jointplot`.
+
+```{important}
+The legends are placed outside the plot if a Figure-level plotting function is used. See scatterplot section below.
+```
+
+Let's now jump into one of the most simple, yet essential, data visualizations: the bar plot.
+
+## Barplots
+
+A good use of a bar plot might be to show counts of something, while poor use of a bar plot might be to show group means. Numerous studies have discussed inappropriate uses of bar plots, noting that "because the bars always start at zero, they can be misleading: for example, part of the range covered by the bar might have never been observed in the sample." Despite the numerous reports on incorrect usage, bar plots remain one of the most common problems in data visualization.
```

Comment:
I'm not sure if this is necessary because it would be explained in the previous `What Makes a Visualization Good` and `Types of Visualization` section of the course. I think we can omit this and focus more so on how to implement bar plots with Seaborn.

Let's ask @lfunderburk for input when she's back from her break.

Reply:
Agreed with removing this section

Reply:
Section removed, ready to resolve comment